analysis_driver/dataset_scanner.py

```diff
@@ -48,18 +48,18 @@ def _get_dataset_records_for_statuses(self, statuses):
             statuses.remove(DATASET_NEW)
             statuses.append(None)
         if len(statuses) > 1:
-            match = {'$or': [{'proc_status': status} for status in statuses]}
+            where = {'$or': [{'aggregated.most_recent_proc.status': status} for status in statuses]}
         else:
-            match = {'proc_status': statuses[0]}
+            where = {'aggregated.most_recent_proc.status': statuses[0]}
         return [
-            d for d in get_documents(self.endpoint, match=match, paginate=False, quiet=True)
+            d for d in get_documents(self.endpoint, where=where, all_pages=True, quiet=True, max_results=100)
```

Comment:
Will we still be able to report on sample repeats with max_results=100?

Reply:
Yes rest_communication will get all pages with option `all_pages=True`